docs/ko/docs/advanced/async-tests.md

```diff
@@ -0,0 +1,110 @@
+# 비동기 테스트 코드 작성
+
+이전 장에서  `TestClient` 를 이용해 **FastAPI** 어플리케이션 테스트를 작성하는 법을 배웠습니다.
+지금까지는 `async` 키워드 사용없이 동기 함수의 테스트 코드를 작성하는 법을 익혔지만 비동기 함수를 사용하여 테스트 코드를 작성하는 것도 매우 유용할 수 있습니다.
+
+예를 들면 데이터베이스에 비동기로 쿼리하는 경우,
+FastAPI 어플리케이션에 요청을 보내고 비동기 데이터 베이스 라이브러리를 사용해 백엔드 서버가 데이터베이스에 데이터를 성공적으로 작성했는지 확인하는 경우를 생각해봅시다.
+
+이런 경우의 테스트 코드를 어떻게 비동기로 작성하는지 알아봅시다.
+
+## pytest.mark.anyio
+
+앞에서 작성한 테스트 함수에서 비동기 함수를 호출하고 싶다면, 테스트 코드도 비동기 함수여야합니다.
+AnyIO는 특정 테스트 함수를 비동기 함수로 호출 할 수 있는 깔끔한 플러그인을 제공합니다.
+
+
+## HTTPX
+
+**FastAPI** 애플리케이션이  `async def` 대신 `def` 키워드로 선언된 함수를 사용하더라도, 내부적으로는 여전히 `비동기` 애플리케이션입니다.
+
+`TestClient`는 표준 pytest를 사용하여 비동기 FastAPI 애플리케이션을 일반적인 `def` 테스트 함수 내에서 호출할 수 있도록 내부적으로 마법 같은 처리를 합니다.
+
+그러나 테스트를 비동기로 실행하면 더 이상 테스트 함수 내에서 `TestClient`를 사용할 수 없게 됩니다
+
+
+TestClient는 <a href="https://www.python-httpx.org" class="external-link" target="_blank">HTTPX</a>를 기반으로 하고 있으며, 다행히 이를 직접 사용하여 API를 테스트할 수 있습니다.
+
+## 예시
+
+간단한 예시를 위해 앞선 챕터 [Bigger Applications](../tutorial/bigger-applications.md){.internal-link target=_blank} 와 [Testing](../tutorial/testing.md){.internal-link target=_blank}:에서 다룬 파일 구조와 비슷한 형태를 확인해봅시다.
```

Comment:
```suggestion
간단한 예시를 위해 [Bigger Applications](../tutorial/bigger-applications.md){.internal-link target=_blank} 와 [Testing](../tutorial/testing.md){.internal-link target=_blank}에서 다룬 파일 구조와 비슷한 형태를 확인해봅시다:
```

Please translate the linked titles. Although they still have not been translated.